carbon/assetlist.json

```diff
@@ -245,11 +248,11 @@
           "type": "wrapped",
           "counterparty": {
             "chain_name": "arbitrum",
-            "base_denom": "0xab19bdaeb37242fa0f30486195f45b9cf5361b78",
-            "contract": "0xb1e6f8820826491fcc5519f84ff4e2bdbb6e3cad"
+            "base_denom": "0xAb19BDAEb37242Fa0f30486195f45b9cF5361b78",
```

Comment:
Error: Trace of carbon, cglp.1.19.1698d3 makes invalid reference to arbitrum, 0xAb19BDAEb37242Fa0f30486195f45b9cF5361b78.

It was probably correct before the letter casing was changed